pkgs/tools/video/tsduck/default.nix

```diff
@@ -0,0 +1,64 @@
+{ stdenv, fetchFromGitHub, fetchurl, makeWrapper
+, dtapi, curl, linuxHeaders }:
+
+stdenv.mkDerivation rec {
+  pname = "tsduck";
+  version = "3.18-1361";
```

Comment:
The most current upstream release seems to be  version 3.19-1520, is there any reason why we should init this at an out-of-date version?